typescript/cli/src/config/chain.ts

```diff
@@ -90,6 +90,52 @@ export async function createChainConfig({
     protocol: ProtocolType.Ethereum,
     rpcUrls: [{ http: rpcUrl }],
   };
+  const wantBlockConfig = await confirm({
+    message: 'Do you want to add block config for this chain?',
+  });
+  if (wantBlockConfig) {
+    const blockConfirmation = await input({
+      message:
+        'Enter no. of blocks to wait before considering a transaction confirmed',
```

Comment:
Probably useful to have some validation/reasonsable bounds here

Reply:
Can we give these ranges :- 
confirmations: 0-10
estimateBlockTime: 0-20
reorgPeriod: 0-50